plugins/Referrers/DataTable/Filter/UrlsFromWebsiteId.php

```diff
@@ -31,10 +31,11 @@ public function filter($table)
     {
         // the htmlspecialchars_decode call is for BC for before 1.1
         // as the Referrer URL was previously encoded in the log tables, but is now recorded raw
-        $table->queueFilter('ColumnCallbackAddMetadata', array('label', 'url', function ($label) {
+        $table->filter('ColumnCallbackAddMetadata', array('label', 'url', function ($label) {
             return htmlspecialchars_decode($label);
         }));
-        $table->queueFilter('ColumnCallbackReplace', array('label', 'Piwik\Plugins\Referrers\getPathFromUrl'));
+        $table->filter('ColumnCallbackReplace', array('label', 'Piwik\Plugins\Referrers\getPathFromUrl'));
+        $table->filter('GroupBy', array('label'));
```

Comment:
Just wondering, if the GroupBy is done before ColumnCallbackAddMetadata (w/ a reduce function specified as `Piwik\Plugins\Referrers\getPathFromUrl`), then maybe `ColumnCallbackAddMetadata` could b queueFilter and we could get rid of the extra `ColumnCallbackReplace`. Does that sound right to you?